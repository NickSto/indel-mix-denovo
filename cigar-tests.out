
M01368:8:000000000-A3GHV:1:1103:15072:19679
1: 251M

M01368:8:000000000-A3GHV:1:1101:6679:24488
1: 3S248M

M01368:8:000000000-A3GHV:1:2105:12460:13136
2: 4M2I245M
insertion after  5: ATCA{AG}CAGG

M01368:8:000000000-A3GHV:1:1101:10077:7324
2941: 166M1D85M
deletion between 3106 and 3108: CTAC[-]TTCA

M01368:8:000000000-A3GHV:1:1108:18888:4511
16390: 179M62I2M2D2M6S
insertion after  16568: CGAT{GGATCACAGGTCTATCACCCTATTAACCACTCACGGGAGCTCTCCATGCATTTGGTATTTTC}GTCT
deletion between 16570 and 16573: TCGT[--]CTGG

M01368:8:000000000-A3GHV:1:1101:20741:16339
1: 199M1D2M2I8M2D2M38S
deletion between 199 and 201: TGCT[-]AACT
insertion after  202: CTAA{CT}GTCT
deletion between 210 and 213: CTTA[--]TACA

M01368:8:000000000-A3GHV:1:1112:20014:11702
1: 52S199M

M01368:8:000000000-A3GHV:1:2111:22092:13076
1: 10S156M85S

M01368:8:000000000-A3GHV:1:1112:22817:23465
31: 251M

M01368:8:000000000-A3GHV:1:2108:17157:13067
111: 205M46S

M01368:8:000000000-A3GHV:1:1104:17482:12333
199: 112M1I138M
insertion after  310: CCCT{C}CCCC

M01368:8:000000000-A3GHV:1:1101:12899:15963
554: 38S3M3I207M
insertion after  556: GAGA{CAG}CCCC

M01368:8:000000000-A3GHV:1:1107:12635:16957
1785: 116M1D135M
deletion between 1900 and 1902: AAGA[-]CCCC

M01368:8:000000000-A3GHV:1:1101:10617:26546
2526: 11S3M1D237M
deletion between 2528 and 2530: ACAG[-]ATTA

M01368:8:000000000-A3GHV:1:2107:13849:20881
2883: 224M1D26M1S
deletion between 3106 and 3108: CTAC[-]TTCA

M01368:8:000000000-A3GHV:1:1104:16663:10687
4099: 171M1I11M68S
insertion after  4269: AATA{C}TGTC

M01368:8:000000000-A3GHV:1:1112:15168:23819
5975: 10S5M1I1M1I233M
insertion after  5979: AGAG{A}CAGG
insertion after  5980: AGAC{A}GGGA

M01368:8:000000000-A3GHV:1:2104:19187:6991
6109: 66S2M1D9M1I173M
deletion between 6110 and 6112: TGTA[-]TAAG
insertion after  6120: GACA{G}TCAT

M01368:8:000000000-A3GHV:1:1106:13629:12848
6800: 3S7M1D1M1D240M
deletion between 6806 and 6808: GACA[-]GGCA
deletion between 6808 and 6810: ACAG[-]GCAT

M01368:8:000000000-A3GHV:1:2109:11237:25755
7603: 12S2M1I1M2D235M
insertion after  7604: GGAG{A}TGTG
deletion between 7605 and 7608: AGAT[--]GTGT

M01368:8:000000000-A3GHV:1:1102:27213:21415
9931: 242M1I3M2D2M3S
insertion after  10172: CGAG{C}TGTC
deletion between 10175 and 10178: CTGT[--]CTCT

M01368:8:000000000-A3GHV:1:1105:20381:18275
10517: 32S6M1D4M1D2M1D207M
deletion between 10522 and 10524: TAAG[-]AGAC
deletion between 10527 and 10529: AGAC[-]AGAT
deletion between 10530 and 10532: ACAG[-]ATAT

M01368:8:000000000-A3GHV:1:2102:23427:17782
11127: 222M3I6M2I5M13S
insertion after  11348: TGAC{CTG}TCTC
insertion after  11354: TCTT{AT}ACAC

M01368:8:000000000-A3GHV:1:2111:6146:18232
13388: 218M2I9M1I1M2D11M9S
insertion after  13605: TAGC{TG}TCTC
insertion after  13614: TATA{C}ACAT
deletion between 13615 and 13618: TACA[--]CATC

M01368:8:000000000-A3GHV:1:1114:11462:20755
14640: 241M2I3M2D5M
insertion after  14880: AAAT{AA}CACC
deletion between 14883 and 14886: ACAC[--]CAGG

M01368:8:000000000-A3GHV:1:1112:11649:18276
16365: 205M39I4M2I1M
insertion after  16569: GATG{GATCACAGGTCTATCACCCTATTAACCACTCACGGGAGC}TCTC
insertion after  16573: TCTC{CA}T
