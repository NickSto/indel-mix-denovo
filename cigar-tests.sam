@HD	VN:1.0	SO:coordinate
@SQ	SN:chr1	LN:249250621
@SQ	SN:chr2	LN:243199373
@SQ	SN:chr3	LN:198022430
@SQ	SN:chr4	LN:191154276
@SQ	SN:chr5	LN:180915260
@SQ	SN:chr6	LN:171115067
@SQ	SN:chr7	LN:159138663
@SQ	SN:chr8	LN:146364022
@SQ	SN:chr9	LN:141213431
@SQ	SN:chr10	LN:135534747
@SQ	SN:chr11	LN:135006516
@SQ	SN:chr12	LN:133851895
@SQ	SN:chr13	LN:115169878
@SQ	SN:chr14	LN:107349540
@SQ	SN:chr15	LN:102531392
@SQ	SN:chr16	LN:90354753
@SQ	SN:chr17	LN:81195210
@SQ	SN:chr18	LN:78077248
@SQ	SN:chr19	LN:59128983
@SQ	SN:chr20	LN:63025520
@SQ	SN:chr21	LN:48129895
@SQ	SN:chr22	LN:51304566
@SQ	SN:chrX	LN:155270560
@SQ	SN:chrY	LN:59373566
@SQ	SN:chrM	LN:16569
@SQ	SN:pUC18	LN:2686
@SQ	SN:phiX174	LN:5386
@RG	ID:R19S5	LB:holland	PL:ILLUMINA	SM:R19S5
M01368:8:000000000-A3GHV:1:1103:15072:19679	99	chrM	1	60	251M	=	105	354	GATCACAGGTCTATCACCCTATTAACCACTCACGGGAGCTCTCCATGCATTTGGTATTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCGAGACGCTGGAGCCGGAGCACCCTATGTCGCAGTATCTGTCTTTGATTCCTGCCCCATCCTATTATTTATCGCACCTACGTTCAATATTACAGACGAACATACTTACTAAAGTGTGTTAATTAATTAATGCTTGTAAGACATAATAATAACAATTGAATG	ABBBBFFFBFFFGGGGGGGGGGHHHHHHGHHHGHGGGGDHHGHHHHHGHHGHHHHHHHHHHGHGHGHHGGGGGGGHGFGGGGGADEGHHHHGGCGGGGGGCHHHGG@@GGGGGGGBFFGFGGGGFGGGFGGFFFFFFFFFFFFFFFFDFFFF/BFFFFFFFFFFFBBFBFDFFFFFFFFFBB//:9;-BBBFFFFF/9B/FFFFFFFFFFFFBFFFFFFFFFFEBFFFFFFFFFFFFFFFFFFFFFFFE/9	NM:i:4	AS:i:231	XS:i:100	RG:Z:R19S5
M01368:8:000000000-A3GHV:1:1101:6679:24488	99	chrM	1	60	3S248M	=	104	338	ATGGATCACAGGTCTATCACCCTATTAACCACTCACGGGAGCTCTCCATGCATTTGGTATTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCGAGACGCTGGAGCCGGAGCACCCTATGTCGCAGTATCTGTCTTTGATTCCTGCCCCATCCTATTATTTATCGCACCTACGTTCAATATTACAGACGAACATACTTACTAAAGTGTGTTAATTAATTAATGCTTGTAAGACATAATAATAACAATTGA	BBCCCFFFFFFCFGGGGFGGGGHHHHHHHHHGHHHHHGGGGGHHHHHHGHHGHHHHHEHHHHHHGHGHGHHGGC@DDDHHGGGGGGGGGHHHHHGGGGGGGGGGHGHGGGFGGGGGGGGGGGGGGFGGGFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFEB/9;:BFFFFFFFFFFFFFFFFFFFFFFEFFFFFEFFFFFFFFFFFF	NM:i:4	AS:i:228	XS:i:103	RG:Z:R19S5
M01368:8:000000000-A3GHV:1:2105:12460:13136	147	chrM	2	60	4M2I245M	=	6	-245	ATCAAGCAGGTCTATCCCCCTATTAACCCCTCACGGGAGCTCTCCATGCATTTGGTATTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCGAGACGCTGGAGCCGGAGCACCCTATGTCGCAGTATCTGTCTTTGATTCCTGCCCCATCCTATTATTTATCGCACCTACGTTCAATATTACAGACGAACATACTTACTAAAGTGTGTTAATTAATTAATGCTTGTAAGACATAATAATAACAATTGAAT	0.00;9/0BBB9/:::.ABF:0B:-A:;.A?A;.EGFB.C9/0GGFFFFGGGFFBBBCC-@@CADEEAGDGHHGGGGEGGCHFHHHHFGGGHHEGFGFHHGGCFGGHHFDEFFHHHFGGGGHHHHHHGHHBHBGFHHHHGGGGGGHEFGDHHHHHHHHGGHGGHHHFGHHHHGFEHHHHHHHGHGGGHHHHHGIGGHHHHHHHGHHHHHHHHHHHHHHHGHHHHHHHHHGGGGGGGGGGFFFFFFFBCCCC	NM:i:8	AS:i:215	XS:i:91	RG:Z:R19S5
M01368:8:000000000-A3GHV:1:1101:10077:7324	99	chrM	2941	60	166M1D85M	=	2946	257	GCGCAATCCTATTCTAGAGTCCATATCAACAATAGGGTTTACGACCTCGATGTTGGATCAGGACATCCCAATGGTGCAGCCGCTATTAAAGGTTCGTTTGTTCAACGATTAAAGTCCTACGTGATCTGAGTTCAGACCGGAGTAATCCAGGTCGGTTTCTATCTACTTCAAATTCCTCCCTGTACGAAAGGACAAGAGAAATAAGGCCTACTTCACAAAGCGCCTTCCCCCGTAAATGATATCATCTCAAC	ABBCCCCFFFFFGGGGGGGGGGHHHHHHHHHHHHHHHHHGHHGGGGGHGGHHHHHHHHHHHHHHHGHHHHHHHHHHHHHHHGGGGGHHHHHHHHHGGHGGHHHHHHGGHHGHHHHHHHHHGHHHHHHGHHGHHHGHHHGGGCEHHHHHHHHFHGGGGGHHHHHHHHHHHHHHHHHHGHHGHHHHHHEHGGHHGHHHHEGHHHHGEHG9FEGGGGGGGGGGGGGGGGGGGGGFFFFFFEFFFFBBBBFFFE/	NM:i:2	AS:i:239	XS:i:112	RG:Z:R19S5
M01368:8:000000000-A3GHV:1:1108:18888:4511	99	chrM	16390	60	179M62I2M2D2M6S	=	16438	180	GGTCCCTTGACCACCATCCTCCGTGAAATCAATATCCCGCACAAGAGTGCTACTCTCCTCGCTCCGGGCCCATAACACTTGGGGGTAGCTAAAGTGAACTGTATCCGACATCTGGTTCCTACTTCAGGGTCATAAAGCCTAAATAGCCCACACGTTCCCCTTAAATAAGACATCACGATGGATCACAGGTCTATCACCCTATTAACCACTCACGGGAGCTCTCCATGCATTTGGTATTTTCGTCTGGGGGG	ABACCFFFFFFFGGGGGGGGGGGHGGGHHHGHHHHHHHGGGGGHHHHHHHHHHHHHHHHHGGFHGGGGGGGGHHHHHHHHHHHGGFGFHHHHEHGGHHHHHHHHHHGGGGGHGHHGGHHHHHHHHHGHHFHHHHHHHHHHHGHHGHHHHHGGGGHGGHHHHGCGGDGHHGFHGHHHGGHGGHGH./GHG:CHHGFHGGGGFGGG0;FFGGBFFGGGBGGGFFFFFEFEFFFFFF./99B//@.9:A:B-=-	NM:i:0	AS:i:180	XS:i:87	RG:Z:R19S5	SA:Z:chrM,1,+,180S71M,60,0;
M01368:8:000000000-A3GHV:1:1101:20741:16339	99	chrM	1	60	199M1D2M2I8M2D2M38S	=	1	201	GATCACAGGTCTATCACCCTATTAACCACTCACGGGAGCTCTCCATGCATTTGGTATTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCGAGACGCTGGAGCCGGAGCACCCTATGTCGCAGTATCTGTCTTTGATTCCTGCCCCATCCTATTATTTATCGCACCTACGTTCAATATTACAGACGAACATACTTGCTAACTGTCTCTTATACACATCTCCGAGCCCACGAGACGGACTCCTATCTCGTA	CCCCCFFFCFFFGGGGGGGGGGHHHHHHGHHHHHGGGGGHHHHHHHHHHHHHHHHHHHHHHGHGHGHHGGGGGGGHHHGGGGGGGHHHHHHGGGGGGGGGGHHHGGGGGGGGGGGGGGGGGGGGEGGGGGGFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFAFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFACCFFFFFFFAFFFFFFFFFFFFFFFE.	NM:i:3	AS:i:181	XS:i:99	RG:Z:R19S5
