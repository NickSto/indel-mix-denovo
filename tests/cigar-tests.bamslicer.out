chrM 5 I
  13136
chrM 199 D
  16339
chrM 199 I
chrM 3106 D
  20881
  7324
chrM 6110 D
  6991
chrM 16568 I
  4511
